DTXMania/Code/Stage/04.Config/CActConfigList.cs

```diff
@@ -621,25 +609,25 @@ public void tSetupItemList_Drums()
                 "HID-SUD:HIDDEN、SUDDEN\n" +
                 "の両方が適用されます。\n" +
                 "STEALTH:チップがずっと表示されません。",
-                "Hidden: Chips disappear at mid-screen\n" +
-                "Sudden: Chips appear from mid-screen\n" +
+                "Hidden: Chips disappear mid-screen\n" +
+                "Sudden: Chips appear mid-screen\n" +
                 "HidSud: Chips appear only at mid-screen\n" +
                 "Stealth: Chips are transparent",
                 new string[] { "OFF", "Hidden", "Sudden", "HidSud", "Stealth" });
             this.listItems.Add(this.iDrumsHIDSUD);
 
             //----------DisplayOption----------
 
-            this.iDrumsDark = new CItemList("       Dark", CItemBase.EPanelType.Normal, (int)CDTXMania.ConfigIni.eDark,
+            this.iDrumsDark = new CItemList("Dark", CItemBase.EPanelType.Normal, (int)CDTXMania.ConfigIni.eDark,
                 "レーン表示のオプションを\n"+
                 "まとめて切り替えます。\n" +
                 "HALF:レーンが表示されなく\n"+
                 "なります。\n" +
                 "FULL:さらに小節線、拍線、\n"+
                 "判定ラインも表示されなくなります。",
-                "OFF: all display parts are shown.\n"+
-                "HALF: lanes and gauge are hidden.\n"+
-                "FULL: additionaly to HALF, bar/beat lines, hit bar are hidden.",
+                "OFF: All display parts are shown.\n"+
+                "HALF: Lanes and gauge are hidden.\n"+
+                "FULL: Same as HALF, with bar/beat lines and hit bar hidden.",
```

Comment:
```suggestion
                "FULL: Same as HALF, with bar/beat lines and judgement line hidden.",
```
Self-editing here: I double-checked the Japanese text and it is judgement line